en/orm/saving-data.rst

```diff
@@ -770,21 +774,20 @@ The ``save()`` method will return the modified entity on success, and ``false``
 on failure. You can disable rules and/or transactions using the
 ``$options`` argument for save::
 
-    // In a controller or table method.
+    // In a controller or table method
     $articles->save($article, ['checkRules' => false, 'atomic' => false]);
 
 Saving Associations
 -------------------
 
-When you are saving an entity, you can also elect to save some or all of the
-associated entities. By default all first level entities will be saved. For
-example saving an Article, will also automatically update any dirty entities
-that are directly related to articles table.
+When you are saving an entity, by default all associated entities,
+that are marekd dirty will be saved. You can however elect to save
+no associated entities by setting the ``associated`` option to ``false``.
```

Comment:
1. So what is right, the API (all levels) or the book (first level only)?
2. *Integer means you would not need to know which ones, just how deep. true/1 would be the default*. Anyway I'd prefer if it worked like described in the API, e.g. it saves by default all nested levels (that were first declared/setup when using newEntity/patchEntity).